### DISCUSSION TITLE
Android Fullscreen?

### DISCUSSION THREAD

**Discussion by nico-5mins** [TheWidlarzGroup/react-native-video#3545] [Ideas]
### What problem are you trying to solve?

From what I have seen in the [npm documentation](https://www.npmjs.com/package/react-native-video#fullscreen), there's no full screen mode available for Android.

Is there a plan for making it work in that OS soon?

### Proposed solution

Add a full screen mode for Android, equivalent to what is available on iOS.

### Platforms

Android

### Additional context

![image](https://github.com/react-native-video/react-native-video/assets/145137942/6929d150-34f3-4328-a3f9-fda67a6f0de4)

**Comment by KrzysztofMoch** [comment]
Hey, docs on npm are for 5.2.1 - I recommend you to try v6 beta (it works better in my opinion). Also you can checkout docs for v6 https://react-native-video.github.io/react-native-video/component/props#fullscreen

**Comment by nico-5mins** [comment]
Could you share a screenshot of how it looks on Android? Cause I have read in different posts and forums that the Android version is lacking quite some features compared to iOS one (like here https://prototyp.digital/blog/react-native-video-handling-fullscreen-and-controls-on-android)

**Reply by KrzysztofMoch** [reply]
Okay, but i will back to you on Monday

**Reply by nico-5mins** [reply]
Hi @KrzysztofMoch 

Could you please send me some screenshots of how that Android player looks like on full screen? A screen recording would be welcomed too!

Thank you

**Reply by gkueny** [reply]
Hi @nico-5mins, you can use one of my repo reproduction to experiment & see by yourself.

https://github.com/gkueny/FullScreenModeTurnOffAndroidReproduction for exemple.

I think you also can use repo exemple https://github.com/react-native-video/react-native-video/tree/master/examples

**Reply by nico-5mins** [reply]
I don't really have access to a coding environment since I am mostly managing the project without hands-on work.

@gkueny I would appreciate it if you could send me any screen recording so that I can take evaluate if it's what we are looking for. Thanks!

**Reply by gkueny** [reply]
Sorry @nico-5mins , do not really have time to re-setup & do the work.
I think you can ask to dev that work on your project

**Reply by nico-5mins** [reply]
Understood, thank you!

**Comment by freeboub** [comment]
fullscreen mode has been implemented, but it supports automatically enable embed controls, like on ios.
